cli/tests/e2e/snapshots/test_ci/test_full_run/autofix-azure-pipelines/meta.json

```diff
@@ -19,5 +19,10 @@
     "scan_environment": "azure-pipelines",
     "is_full_scan": true,
     "is_sca_scan": false
-  }
+  },
+  "policy": [
```

Comment:
Oh yeah I wanted to ask y'all about that! This got added to meta when I did a snapshot-update and I had no idea why 😅  Can you shed some light on what this is for and how to make it work without it?

Reply:
It looks like you added it here: https://github.com/returntocorp/semgrep/pull/5928/files#diff-7c7cda58065ec85eebe55f7c9b79daed7e22b02b2ee553bb3379317a0ec4e505R162

Reply:
Oh no that's different.

Reply:
No I take it back it's definitely that line. Looks like you are sending this info in the body of the request?

Reply:
Oh, yes we are. And we need to send that info in the body of the request so that the scan object on semgrep-app has that information.

Reply:
This tells semgrep-app which policies were used to perform the scan. As we work to make semgrep ci work even if/when semgrep-app is down, we needed to separate all "get" logic from all "post" logic. While the "post" _could_ look up what policies are attached to the repo in question, that information could be out-of-date by the time the POST actually goes through.

Reply:
Wouldn't the info sent by cli be strictly more stale than doing a lookup on the backend? Also for 99% of users wont this always be block,comment,audit?

Reply:
It will be more stale in terms of what the user has configured, but possibly inaccurate in terms of recording what rules/policies the scan actually used, since the scan is using the rules that came from the initial GET request.